PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+# PULL REQUEST TEMPLATE
```

Comment:
You know, that you can add also PR templates under .github. Just search in Google for github issue and pr templates to learn how. The issue templates you can already find in this folder in the repo!